hugo/layouts/partials/langium-footer.html

```diff
@@ -1,20 +1,30 @@
 <!-- Footer Menu -->
-<footer id="website-footer" class="font-mono block md:flex dark:text-gray-100 justify-center items-center mt-12 mb-6">
-    <div class="hover:underline px-4 block text-left md:text-center m-4" style="width:180px">
-        <a href="https://www.typefox.io/imprint-privacy/#privacy">
+<footer id="website-footer" class="font-mono block md:flex md:flex-wrap dark:text-gray-100 justify-center items-center mt-12 mb-6">
+    <div class="px-4 block text-left md:text-center m-4" style="width:180px">
+        <a href="https://projects.eclipse.org/projects/ecd.langium/" class="hover:underline">
+            About
+        </a>
+    </div>
+    <span class="text-lg hidden md:inline">|</span>
+    <div class="px-4 block text-left md:text-center m-4" style="width:180px">
+        <a href="http://www.eclipse.org/legal/privacy.php" class="hover:underline">
             Privacy Policy
         </a>
     </div>
     <span class="text-lg hidden md:inline">|</span>
-    <div class="hover:underline px-4 block text-left md:text-center m-4" style="width:180px">
-        <a href="https://www.typefox.io/imprint-privacy/#copyright">
-            Copyright
+    <div class="px-4 block text-left md:text-center m-4" style="width:180px">
+        <a href="http://www.eclipse.org/legal/termsofuse.php" class="hover:underline">
+            Terms of Use
         </a>
     </div>
     <span class="text-lg hidden md:inline">|</span>
-    <div class="hover:underline px-4 block text-left md:text-center m-4" style="width:180px">
-        <a href="/imprint">
-            Imprint
+    <div class="px-4 block text-left md:text-center m-4" style="width:180px">
+        <a href="http://www.eclipse.org/legal/copyright.php" class="hover:underline">
+            Copyright Agent
```

Comment:
Why Copyright Agent and not just Copyright like above?
Never heard Agent before but I am sure it makes absolutely sense.
Thx

Reply:
It's the same on other Eclipse Foundation websites.